Lib/idlelib/configdialog.py

```diff
@@ -518,12 +486,17 @@ def create_page_highlight(self):
             'Shell Stderr Text': ('stderr', '13'),
             }
         parent = self.parent
-        self.builtin_theme = StringVar(parent)
-        self.custom_theme = StringVar(parent)
+        self.builtin_theme = tracers.add(
+                StringVar(parent), self.var_changed_builtin_theme)
+        self.custom_theme = tracers.add(
+                StringVar(parent), self.var_changed_custom_theme)
         self.fg_bg_toggle = BooleanVar(parent)
```

Comment:
It is now obvious that fg_bg_toggle is not traced, others are.  A plus for the change.